lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -876,32 +892,53 @@ slotDifference epl a b
     a' = flatSlot epl a
     b' = flatSlot epl b
 
+-- | Return the slot immediately before the given slot.
```

Comment:
Gotta admit that I am not fond of the alignment style below :/ ... Plus it clashes with our policy of no variable-length indentation!

Reply:
> Gotta admit that I am not fond of the alignment style below :/ ... Plus it clashes with our policy of no variable-length indentation!

You're right! I'll remove this indentation.

(I was using this style while writing as it made it visually easier for me to read, but forgot to remove it before pushing.)

Reply:
Fixed.